### ISSUE TITLE
Transferwise - Odoo Server Error: No API key specified!

### ISSUE THREAD

**Issue by alex9434** [OCA/bank-statement-import#304]
I am unable to import bank statements from Transferwise.

1. Set-up Transferwise bank account: TransferWise.com as online bank statements provider in Online Bank Statements (OCA) section
2. Online Statement Pull Wizard: Odoo Server Error: No API key specified!

Where can I set the API-Key to make this work?

System: Odoo 12 OCB on Debian 10 with Conda 3.7.2

**Comment by alexey-pelykh** [comment]
Have you checked the readme? https://github.com/OCA/bank-statement-import/blob/12.0/account_bank_statement_import_online_transferwise/readme/CONFIGURE.rst

**Comment by alex9434** [comment]
Hi @alexey-pelykh, I tried to do that but got stuck. I put in the API key, that I got from Transferwise (i created a read-only API key, is that correct?) but thy system also wants a "Profile" (see screenshot). I have no idea what to put in there. Unfortunately the system does not seem to use the key without the "Profile".
![Screenshot from 2020-06-14 11-38-46](https://user-images.githubusercontent.com/8560446/84584247-612b9500-ae34-11ea-9599-f65047cfb951.png)

**Comment by alexey-pelykh** [comment]
Profile is a required setting, since one API key grants access to both personal and business profiles. If the dropdown is empty, likely API request to retrieve list of profiles failed for some reason, thus a) check logs b) try to reproduce that on runbot to avoid case when it's something wrong with your instance of Odoo

**Comment by alex9434** [comment]
Hi @alexey-pelykh, I have deleted the account and re-done everything. This time I was able to chose in the drop-down box the business account. I tried to pull the statements for Q2 and there is no error message in the log. Thank you so much for your tipps and help!

One thing that I noticed is that the bank statements would only include the currency account and not payments that are doneon the cross-border accounts. Do I need to add multiple accounts in the respective currencies to get these payments or are cross-border accounts currently not supported?
